web-client/tests/wasm.rs

```diff
@@ -66,7 +66,9 @@ pub async fn it_can_initialize_with_mock_network() {
         }),
     );
 
-    spawn_local(consensus);
```

Comment:
Why was this removed?

Reply:
Because the NodeJS test runner was not exiting after the test completed successfully, because of pending timeouts or intervals. We tried to work around it by not spawning the consensus, but only polling it once (the added lines after this removed line), but that didn't work really either.

I have now changed the WASM tests to run in headless Chrome and there the test runner exits correctly, so I have added this back in.

Reply:
After the last rebase of this PR by @styppo the commit reverting this is gone. The relevant changes to the test runner are now in PR #2513. Once that is merged, this PR can be rebased on top and this change here reverted.